### PR TITLE
208004 fix jobs view

### PR DESCRIPTION
**Pull request by nilsgstrabo**
Minor updated to job/batch view related to radix-api changes for jobs and batches.

To run locally, radix-api from branch [208004-radixbatch](https://github.com/equinor/radix-api/tree/208004-radixbatch) must also be running locally.

### CODE REVIEW

**Review by Pespiri — CHANGES_REQUESTED** [review]

File: src/models/scheduled-job-summary/test-data.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export const testData: TestDependencyDataType<ScheduledJobSummaryModel> = [
     backoffLimit: 10,
     resources: ReplicaResourcesData[0],
     node: ReplicaNodeData[0],
+    deploymentName: 'any-deployment',
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/models/scheduled-job-summary/test-data.ts:35]: fixed

File: src/models/scheduled-batch-summary/test-data.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ export const testData: TestDependencyDataType<ScheduledBatchSummaryModel> = [
     replica: ReplicaSummaryData[0] as unknown as ReplicaSummaryNormalizedModel,
     jobList: [ScheduledJobSummaryData[0]],
     totalJobCount: 0,
+    deploymentName: 'any-deployment',
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/models/scheduled-batch-summary/test-data.ts:26]: fixed

File: src/components/component/scheduled-batch-list.tsx [hunk 3]
```diff
@@ -27,6 +26,7 @@ import {
 } from '../../utils/table-sort-utils';
 
 import './style.css';
+import { JobDeploymentLink } from './job-deployment-link';
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/components/component/scheduled-batch-list.tsx]: fixed

File: src/components/component/scheduled-batch-list.tsx [hunk 4]
```diff
@@ -174,19 +177,16 @@ export const ScheduledBatchList = ({
                             />
                           </Table.Cell>
                         </Table.Row>
-                        {expanded && (
+                        {expanded && batch.deploymentName && (
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/components/component/scheduled-batch-list.tsx]: fixed

File: src/components/component/job-deployment-link.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import * as PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/components/component/job-deployment-link.tsx]: fixed

**Review by Pespiri — APPROVED** [review]

**Pull request merged**